app/models/bus_stop.rb

```diff
@@ -7,10 +7,10 @@ class BusStop < ApplicationRecord
   has_paper_trail
 
   has_and_belongs_to_many :routes
-  has_many :bus_stop_fields
+  has_many :bus_stop_fields, dependent: :destroy
   has_many :fields, through: :bus_stop_fields
 
-  accepts_nested_attributes_for :bus_stop_fields
+  accepts_nested_attributes_for :bus_stop_fields, update_only: true
```

Comment:
Where do BSFs get created, if not by virtue of nested attributes?

Reply:
you're right.